package.json

```diff
@@ -59,7 +59,6 @@
     "coffeelint": "<2.0",
     "express": "^4.17.1",
     "jasmine-focused": ">=1.0.7 <2.0",
-    "node-gyp": "^5.1.1",
```

Comment:
Still in the process of trying to fix the spec so that it does a proper reference without the need for hoisting the `node-gyp` package into the package.lock. Which should also have the added benefit (when completed) of being entirely node version agnostic

Reply:
I think we may have a much harder time doing that test with npm 8, which I hope to move to before May, since npm 6 is going EOL in May.

So it may be small returns for fixing this and potentially having to drop the test entirely for npm 8. It's a long story but I can try to explain it if folks want, but basically the npm config `node-gyp`, which gives a path to the copy of node-gyp somewhere on your machine that you want to use, the config is not respected in npm 7, and I don't think it's respected in npm 8 or 9 either?? Would need to test again, my memory is foggy.

But we would basically have to copy-paste (or symlink?) all of ppm (or at least all of npm?) into a new dir with space in the path and run the test that way... Or just drop the test.

An Atom maintainer was leaning toward/comfortable with "just drop it" last I talked to them about it.

Sorry to go on and on, kind of brain dumping about this, feel free to ask questions or do what speaks to you, I want to be more helpful than annoying here. Not trying to impose. Just I have this stuff vaguely in mind for my to-dos already around this repo, but last thing I want to do is put others off contributing, you know?

Reply:
I've gone ahead and made the executive decision to simply comment out that test. If it won't work going forward, then it's just an impedance and I'm comfortable with no longer including it.